site/docs/spark-queries.md

```diff
@@ -220,27 +218,140 @@ SELECT * FROM prod.db.table.files
-   This usually occurs when reading from V1 table, where `contains_nan` is not populated. 
+<div class="markdown-table-container" markdown="block">
+| partition | record_count | file_count |
+| -- | -- | -- |
+|  {20211001, 11}|           1|         1|
+|  {20211002, 11}|           1|         1|
+|  {20211001, 10}|           1|         1|
+|  {20211002, 10}|           1|         1|
+</div>
+
+### All Metadata Tables
+
+These tables are unions of the metadata tables that are specific to a snapshot.
+
+For example, one table has two snapshots, that's mean a snapshot corresponding to a metadata table, and the "all" metadata tables will contain the results of these two metadata tables.
+
+!!! Warning
+    The "all" metadata tables may produce more than one row per data file or manifest file because metadata files may be used in more than one table snapshot.
+
+#### All Data Files
+
+To show a table's valid data files and each file's metadata, run:
+
+```sql
+SELECT * FROM prod.db.table.all_data_files
+```
+
+<div class="markdown-table-container" markdown="block">
+| content | file_path | file_format | partition | record_count | file_size_in_bytes | column_sizes| value_counts | null_value_counts | nan_value_counts| lower_bounds| upper_bounds|key_metadata|split_offsets|equality_ids|sort_order_id|
+| -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- |
+|      0|s3:.../dt=20210102/xxx.parquet|    PARQUET|{20210102}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210102}|{1 -> 2, 2 -> 20210102}|        null|          [4]|        null|            0|
+|      0|s3:.../dt=20210103/xxx.parquet|    PARQUET|{20210103}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210103}|{1 -> 3, 2 -> 20210103}|        null|          [4]|        null|            0|
+|      0|s3:.../dt=20210104/xxx.parquet|    PARQUET|{20210104}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210104}|{1 -> 3, 2 -> 20210104}|        null|          [4]|        null|            0|
+</div>
+
+#### All Manifests
+
+To show a table's valid file manifests and each file's metadata, run:
+
+```sql
+SELECT * FROM prod.db.table.all_manifests
+```
+
+<div class="markdown-table-container" markdown="block">
+| path | length | partition_spec_id | added_snapshot_id | added_data_files_count | existing_data_files_count | deleted_data_files_count| partition_summaries|
+| -- | -- | -- | -- | -- | -- | -- | -- |
+| s3:/.../table/data/dt=20210102/xxx.parquet | 6376 | 0 | 6272782676904868561 | 2 | 0 | 0 |[{false, false, 10, 11}, {false, false, 20210101, 20210101}]|
+</div>
+
+!!! Note
+    1. A valid manifest file is one that is referenced from any snapshot currently tracked by the table.
```

Comment:
Not sure if this was added because another review?  But I'm not sure there's much use in this line and think it can be removed, as this is a common fact across all "all_x" tables, and we already covered it in the opening section of "All Metadata Tables" if we go with my suggestion there.

Reply:
So, i think what you said, is already covered when we changed the other comment on line 266:

> The "all" metadata tables may produce more than one row per data file or manifest file because metadata files may be used in more than one table snapshot.
 
So can we remove this sentence?